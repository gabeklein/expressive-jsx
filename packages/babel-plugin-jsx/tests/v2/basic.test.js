transform("style", `
  const App = () => {
    color: red;

    <this>Hello World</this>
  }
`)

transform("nested style", `
  const App = () => {
    element: {
      color: red;
    }

    <element>Hello World</element>
  }
`)