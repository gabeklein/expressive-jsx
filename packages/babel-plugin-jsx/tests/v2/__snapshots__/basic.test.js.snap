// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`nested style 1`] = `
  const App = () => {
    return <div className="element_52j">Hello World</div>;
  };
  
  css(\`
  	.element_52j {
  		color: red;
  	}
  \`);
`;

exports[`style 1`] = `
  const App = () => {
    return <div className="App_8dv">Hello World</div>;
  };
  
  css(\`
  	.App_8dv {
  		color: red;
  	}
  \`);
`;
